src/components/Footer/Footer.css

```diff
@@ -1,15 +1,18 @@
 .Footer {
```

Comment:
![image](https://user-images.githubusercontent.com/45266424/120916495-2f27ca00-c667-11eb-9ad7-4dd85555beb9.png)
I think there should be a little extra vertical space between the social media icons and the copyright!